libcec/patches/999-compile-with-fpermissive.patch

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
If you want to go for a patch, you should submit this upstream.

Have you tried to add this to the OpenWrt Makefile instead?
TARGET_CXXFLAGS += -fpermissive

Reply:
No, I was not aware of `TARGET_CXXFLAGS` (again: not enough OpenWrt makefile knowledge...). Sounds like a much better solution, if this actually propagates far enough!

I'll try it as soon as I find the time (currently swamped in other things, like making the connectedhomeip/matter gn/ninja build system build for openwrt targets...)

Reply:
Interesting. I have a [draft PR for openthread](https://github.com/openwrt/packages/pull/19218) but haven't looked into matter yet.